apps/contrib/assets/SwitchButton.jsx

```diff
@@ -1,10 +1,14 @@
 import React, { useState } from 'react'
 
 export const SwitchButton = (props) => {
-  const [isChecked, setIsChecked] = useState(props.initiallyChecked || false)
```

Comment:
I changed this to isChecked, otherwise it wouldn't work. I guess adding the `|| false` (or true) also makes sense?